### ISSUE TITLE
Tutorial and pipeline for inference

### ISSUE THREAD

**Issue by malteekj** [OSchoppe/AIMOS#3]
Thanks for the interesting work and paper. 
Can I suggest to update this repository with better documentation for the setup as well as a tutorial and pipeline for inference? I am very interested in using your pretrained networks, but it would be quite difficult to use in its current form.

The data preparation pipeline is quite cumbersome to use. It would be preferable to just load the data directly from Nifty or DICOM rather than generating TIFF files. Also, the resampling step should find the meta data of new incoming data and resample it to the same spatial resolution per pixel rather than just the same amount of pixels. The standardization step is also not commonly performed for CT, as HU is comparable across scans. Rather, windowing and normalization would be better.

I managed to run inference on one scan after some code-parkour, and it didn't really yield anything meaningful. This might be due to problems with using new data with your pipeline, so I hoped you might update the code, so it would be possible to use for the community.

**Comment by malteekj** [comment]
Unfortunately, I haven't been able to get anything meaningful either. I think the model might be functional, if the authors provide more information, such as windowing etc. But as I also described, I don't have complete faith in the model working with new data.

It might be better to train a new network on the data (which is publicly available).

**Comment by ecramer22** [comment]
When I used the publicly available data with a pre-trained model, the results are okay, but no where near as good as the demo shows. I will try know to use this data to train a new network and update you on the results. However, the training process does not seem all that clear. Please let me know if you find anything meaningful in your deep-dive of the code and public data. Here are a few images that I curated using the demo:

![M03DemoDataFullBody](https://user-images.githubusercontent.com/67476365/174153763-702ee1c9-a0f8-4588-871c-878554e52b9b.png)
![M03DemoDataIndiv](https://user-images.githubusercontent.com/67476365/174153764-a4ffbbf9-7194-412c-bb7b-174dd75ac6e0.png)

And here is what I got when using the publicly available LSFM data:

![mouse_5FullBody](https://user-images.githubusercontent.com/67476365/174153839-614c0fd4-c75e-4359-83e2-2b706307bd7d.png)
![mouse_5Indiv](https://user-images.githubusercontent.com/67476365/174153840-fb2229ee-a1b5-405a-a726-d81b6faee249.png)

**Comment by EnriqueFV** [comment]
After fixing some code problems related to the paths, I ran the demonstration locally, but the predicted volume was empty. I double-checked that the model received the sample data (the TIFF images) and the properly trained model (NAC). Still, after all this, I found that the model didn't produce any segmentation of the sample data. Now, after reading your post, it seems doubtful this approach will ever work on my own CT data. 

It's a pity because it really seems useful for the scientific field.
